### PR TITLE
Des 383 be modify user progress

### PR DESCRIPTION
**Pull request by Daniielmayo**
**https://danielcalleco.atlassian.net/browse/DES-383?atlOrigin=eyJpIjoiMTQxNDZiNGJiZDM1NGExOTg2MmZlY2U2MmFkZGFiMTEiLCJwIjoiaiJ9**

### CODE REVIEW

File: controllers/user/coursesProgressUser.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        lastViewedTopic: {
+          topicId: topicId,
+          lastViewedResource: resourceId,
+        },
+      };
+
+      user.CourseProgress.push(courseProgress);
+    } else {
+      // Display the lastViewedTopic stored in the user's progress
+
```
Comment (DanielCalleCO) [inline controllers/user/coursesProgressUser.js:58]: Could you please delete this blank line?

File: routes/user/progressUser.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+const { Router } = require("express");
+const {
+  CourseProgress,
+} = require("../../controllers/user/coursesProgressUser");
+
+const router = Router();
+
+/**
```
Comment (DanielCalleCO) [inline routes/user/progressUser.js:8]: Could you please add a line explaining this commented section?

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]

File: controllers/user/coursesProgressUser.js (hunk 1 above)
Comment (Daniielmayo) [reply controllers/user/coursesProgressUser.js:58]: line delete

File: routes/user/progressUser.js (hunk 2 above)
Comment (Daniielmayo) [reply routes/user/progressUser.js:8]: This is not a comment, it is the implementation of the swagger tool to document the back-end

**Review by abdulflorez — APPROVED** [review]

**Review by generaciondesafiante — APPROVED** [review]

**Pull request merged**